db/migrate/20230329174200_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :Text
```

Comment:
- Hi :clap: , the text should be text and in lowercase ` t.text :text`, let's fix this, please.